packages/svelte/src/internal/client/runtime.js

```diff
@@ -624,23 +618,21 @@ function process_effects(effect, collected_effects) {
 
 	main_loop: while (current_effect !== null) {
 		var flags = current_effect.f;
-		var is_active = (flags & INERT) === 0;
 		var is_branch = (flags & BRANCH_EFFECT) !== 0;
-		var is_clean = (flags & CLEAN) !== 0;
-		var child = current_effect.first;
+		var is_skippable_branch = is_branch && (flags & CLEAN) !== 0;
```

Comment:
we could do this instead which would presumably be a tiny bit faster, but less readable

```suggestion
		var is_skippable_branch = (flags & (CLEAN | BRANCH_EFFECT)) === (CLEAN | BRANCH_EFFECT);
```